firmware/aek/config.h

```diff
@@ -23,8 +23,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define VENDOR_ID    0x1209
 #define PRODUCT_ID   0x9209
 #define DEVICE_VER   0x0001
-#define MANUFACTURER djvs
-#define PRODUCT      aek
+#define MANUFACTURER ALPS
+#define PRODUCT      Apple Extended Keyboard 2
```

Comment:
The product/device ID correspond to a USB vendor & PID I obtained (1209 9209) and are specific to the board, please don't change these values.

Reply:
reverted changes